rfcs/xxxx-standard-gh-labels.md

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
I'd say number it 0003 as that would be the next sequential one. And, even if we reject, still good to keep it numbered in some fashion I think?

Reply:
Works for me